src/routes/privateSnotra.ts

```diff
@@ -0,0 +1,54 @@
+import { FastifyInstance } from 'fastify';
+import createOrGetConnection from '../db';
+
+export default async function (fastify: FastifyInstance): Promise<void> {
+  fastify.get('/all_tags', async (req, res) => {
+    if (!req.service) {
+      return res.status(400).send();
+    }
+
+    const con = await createOrGetConnection();
+    const result = await con.query(`
+       SELECT value AS name
+       FROM keyword
+       WHERE status = 'allow'
+       ORDER BY name ASC
+    `);
```

Comment:
We don't really use the query command on API side, prefer to use the build in query builder, examples is the `getSquads` function in boot for instance.